teps/0091-trusted-resources.md

```diff
@@ -312,6 +316,11 @@ type VerificationPolicySpec struct {
 	Resources []ResourcePattern `json:"resources"`
 	// Authorities defines the rules for validating signatures.
 	Authorities []Authority `json:"authorities"`
+	// Mode controls whether a failing policy will be rejected (not admitted), or if errors are converted to Warnings.
+	// enforce - Reject (default)
+	// warn - allow but warn
+	// +optional
+	Mode string `json:"mode,omitempty"`
```

Comment:
nit: i'm confused by the text a few lines up "(Reference from [policy-controller](https://github.com/sigstore/policy-controller))" - is that meaning to say that this code is copied directly from the policy-controller as an example of what the API could be, or is it saying that the code here is inspired by the policy-controller?

If it's the latter (and i think it is?) it might be clearer to say something like "inspired by the policy-controller" or "based on the policy-controller" or have a separate section with something like "similar approaches in other projects"

Reply:
Thanks! It is the latter. Will update

Reply:
Updated it and also the docstrings.